burn-book/src/basic-workflow/training.md

```diff
@@ -22,7 +22,7 @@ impl<B: Backend> Model<B> {
         targets: Tensor<B, 1, Int>,
     ) -> ClassificationOutput<B> {
         let output = self.forward(images);
-        let loss = CrossEntropyLoss::new(None).forward(output.clone(), targets.clone());
+        let loss = CrossEntropyLoss::new(None, &output.device().forward(output.clone(), targets.clone());
```

Comment:
I think you are missing a closing ) before the `forward`:

`CrossEntropyLoss::new(None, &output.device()).forward(output.clone(), targets.clone());`

Reply:
You're right.  Should be fixed now.